homeassistant/components/notify/hangouts.py

```diff
@@ -59,7 +60,8 @@ def send_message(self, message="", **kwargs):
         messages.append({'text': message, 'parse_str': True})
         service_data = {
             ATTR_TARGET: target_conversations,
-            ATTR_MESSAGE: messages
+            ATTR_MESSAGE: messages,
+            ATTR_DATA: kwargs.get(ATTR_DATA, {}) or {}
```

Comment:
Put defaults in the service schema, ie put the empty dict there instead. Here just do:
```py
ATTR_DATA: kwargs[ATTR_DATA]
```